framework/src/engine/rpc/rpc_server.ts

```diff
@@ -305,4 +314,12 @@ export class RPCServer {
 		}
 		return existingMethod;
 	}
+
+	private _isBlacklistedMethod(namespace: string, method: string): boolean {
+		const { blacklist } = this._config;
+		if (!blacklist) {
+			return false;
+		}
+		return blacklist.includes(namespace) || blacklist.includes(`${namespace}_${method}`);
```

Comment:
`blacklist.includes(namespace)` might ban non intended namespace. ex) some method includes part of name space or namespace includes partial namespace

Reply:
this comparison was correct for array